config.w32

```diff
@@ -29,7 +29,7 @@ if (PHP_MONGO != "no") {
   }
 
   if (CHECK_LIB("ssleay32.lib", "mongo", PHP_MONGO) && CHECK_LIB("libeay32.lib", "mongo", PHP_MONGO)) {
-    ADD_FLAG("CFLAGS_MONGO", " /D HAVE_MONGO_SSLLIB=1");
+    ADD_FLAG("CFLAGS_MONGO", " /D HAVE_MONGO_OPENSSL=1");

```

Comment:
`HAVE_MONGO_SSLLIB` was originally added in https://github.com/mongodb/mongo-php-driver-legacy/commit/ec3b84c78f4dcb09d5b20b6d23736357daf5d305 but never used in the source. Changing this to conditionally define `HAVE_MONGO_OPENSSL` makes the Windows build consistent with `config.m4` calling `PHP_SETUP_OPENSSL()` (and is actually what `SETUP_OPENSSL()` does in newer PHP versions' `configure.js`).

Note that even if we don't find the libraries here, the driver can still establish SSL connections via PHP's streams API. That depends entirely on whether the OpenSSL extension is loaded. If we don't find the libraries here, that only means that `X509`, `GSSAPI` and `PLAIN` auth mechanisms will not be supported. In practice, PECL's build system should always find the libraries so even that case should never come up.